scripts/dotnet-cli-build/CompileTargets.cs

```diff
@@ -203,6 +207,25 @@ private static BuildTargetResult CompileStage(BuildTargetContext c, DotNetCli do
             File.Delete(Path.Combine(binDir, $"dotnet{Constants.ExeSuffix}"));
             File.Copy(Path.Combine(binDir, $"corehost{Constants.ExeSuffix}"), Path.Combine(binDir, $"dotnet{Constants.ExeSuffix}"));
 
+            // HACK
+            // bootstrapping for Windows x86. Copy csc/vbc from stage0.
+            // This is a temporary hack for https://github.com/dotnet/roslyn/issues/8951
+            if (IsWinx86)
```

Comment:
#1504 With Conditional Targets went in recently.

I'm not convinced it's the fix all, as some of these logic forks are pretty small and may not require a separate target, but it's bound to provide some relief here.